### PR TITLE
Add metadata fields to package.json and refactor in readme file

### PR DESCRIPTION
**Pull request by chhakuli123**
### Description
Closes: #9 
This PR adds the following metadata fields to the package.json file and refactor the readme file:
- description: A brief description of the project.
- keywords: Keywords to improve search visibility.
- author: The author's name and email contact.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,52 @@
 - 📊 Similarity percentage calculation
 - 🔧 Easy to integrate
   
+## Basic Usage 🚀
+
```
Comment (atapas) [inline README.md:31]: Add

npm install text-compare

**Review by atapas — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -1,6 +1,19 @@
...
+    "diff",
+    "react",
+    "typescript"
+  ],
+  "author": {
+    "name": "CreoWis",
+    "email": "hello@creowis.com",
+    "url": "https://www.creowis.com/"
+  },
   "type": "module",
```
Comment (atapas) [inline package.json:17]: Add repository information

**Review by atapas — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (chhakuli123) [reply package.json:25]: Done

File: README.md (hunk 1 above)
Comment (chhakuli123) [reply README.md:31]: Done

**Review by atapas — APPROVED** [review]

**Pull request merged**